consult-dir.el

```diff
@@ -93,12 +97,29 @@
 
 (defcustom consult-dir-shadow-filenames t
   "Shadow file names instead of replacing them when using `consult-dir'."
+  :group 'consult-dir
+  :type 'boolean)
+
+(defcustom consult-dir-include-tramp-hosts t
+  "Whether or not to include remote hosts when using `consult-dir'."
+  :group 'consult-dir
   :type 'boolean)
 
 (defcustom consult-dir-default-command #'find-file
   "Default command to run after selecting a directory using `consult-dir'."
+  :group 'consult-dir
   :type 'function)
 
+(defcustom consult-dir-default-remote-path "~"
```

Comment:
Rename to `consult-dir-tramp-path-default` or `consult-dir-tramp-remote-path-default`. Basically try to preserve the `consult-dir-tramp-` namespace for tramp details.

Reply:
I do usually, this one was just missed. Thanks for catching